receiver/jaegerreceiver/README.md

```diff
@@ -77,7 +77,7 @@ receivers:
     protocols:
       grpc:
     remote_sampling:
-      fetch_endpoint: "jaeger-collector:1234"
+      host_endpoint: "localhost:1234"
```

Comment:
I'd keep the `jaeger-collector` as the hostname, as it conveys clearly which hostname is expected here.

Reply:
Done.